wallet.go

```diff
@@ -722,6 +808,22 @@ func (w *Wallet) AddressUsed(addr waddrmgr.ManagedAddress) bool {
 	return addr.Used()
 }
 
+// AccountUsed returns whether there are any recorded transactions spending to
+// a given account. It returns true if atleast one address in the account was used
+// and false if no address in the account was used.
+func (w *Wallet) AccountUsed(account uint32) (bool, error) {
+	addrs, err := w.Manager.ActiveAccountAddresses(account)
```

Comment:
This is an assumption because currently ActiveAccountAddresses returns *all* addresses from the account, but may not in the future.  If it is changed (when single use addrs are added) to only return unused addresses, the used account may be marked unused.